api/v1alpha1/database_types.go

```diff
@@ -28,8 +28,13 @@ import (
 // DatabaseSpec defines the desired state of Database
 type DatabaseSpec struct {
 	// +kubebuilder:validation:Required
-	NDBRef   string   `json:"ndbRef"`
-	Instance Instance `json:"databaseInstance"`
+	NDBRef string `json:"ndbRef"`
+	// +optional
+	IsClone bool `json:"isClone"`
```

Comment:
Should IsClone be made mandatory? This is so the user explicitly knows whether does this input spec belongs to clone type or new provisioning type. I think we had initially set a param called "[provisionType](https://github.com/nutanix-cloud-native/ndb-operator/pull/119/files#diff-6c04d9bdef018e61ac8ffcdb2d1dad7d67094708bb24f09dc3a7221110ae7ab7R34)"

Reply:
I'm fine with either. I feel like it's nice from a usability standpoint to default to isClone equals false (which is what we are currently doing), as I imagine the most common usecase would be provisoining databases. It would be annoying for the user to have to maintain setting isClone to be true, what are your thoughts @manavrajvanshi ?

Reply:
Can also run this by Jeremy for product-side input

Reply:
I think the default for isClone would be good. Lesser inputs for devs. Since the demo has been recorded with this functionality which will be presented at Kubecon soon, I would refrain from changing this in any case until the demo is done. 
We can run it by Jeremy and track this for a later upgrade. Let me know what you guys think @mazin-s @akshmish @yash-ntnx

Reply:
Agreed (I believe Akshay and I also discussed that since the demo was recorded we shouldn't change it anyway).

Reply:
I'd prefer to default isClone to false. Thats how NDB's APIs also behave